docs/document/content/user-manual/shardingsphere-scaling/build.cn.md

```diff
@@ -18,7 +18,61 @@ mvn clean install -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Drat.skip=tr
 
 或者通过[下载页面]( https://shardingsphere.apache.org/document/current/cn/downloads/ )获取安装包。
 
-2. 解压缩 proxy 发布包，修改配置文件 `conf/server.yaml`，这里主要是开启 `scaling` 和 `mode` 配置：
+> Scaling还是实验性质的功能，在持续进化中，建议使用master分支最新版本，点击此处[下载最新版本]( http://117.48.121.24:8080/Proxy/ )
+
+2. 解压缩 proxy发布包，修改配置文件`conf/sharding.yaml`，配置`schemaName`，`dataSources`，`rules`里的`shardingAlgorithms`和`keyGenerators`，以下示例可供参考：
+```yaml
+schemaName: sharding_db
+
+dataSources:
+ ds_0:
+   url: jdbc:mysql://127.0.0.1:3308/account?serverTimezone=UTC&useSSL=false
+   username: root
+   password: root
+   connectionTimeoutMilliseconds: 30000
+   idleTimeoutMilliseconds: 60000
+   maxLifetimeMilliseconds: 1800000
+   maxPoolSize: 50
+   minPoolSize: 1
+
+rules:
+- !SHARDING
+ tables:
+   user:
+     actualDataNodes: ds_0.user
+     databaseStrategy:
+       standard:
+         shardingColumn: id
+         shardingAlgorithmName: database_inline
+     tableStrategy:
+       standard:
+         shardingColumn: id
+         shardingAlgorithmName: user_inline
+     keyGenerateStrategy:
+       column: id
+       keyGeneratorName: snowflake
+ 
+ shardingAlgorithms:
+   database_inline:
+     type: INLINE
+     props:
+       algorithm-expression: ds_0
+   user_inline:
+     type: INLINE
+     props:
+       algorithm-expression: user_${id % 16}
+ 
+ keyGenerators:
+   snowflake:
+     type: SNOWFLAKE
+     props:
+       worker-id: 123
+```
```

Comment:
Could we just put a relative link here? which explain how to configure`config-sharding.yaml`.

Reply:
yes, I will change this.